### PR TITLE
Update RAC to 4.0-alpha.3

### PR DESCRIPTION
**Pull request by ikesyo**

### CODE REVIEW

File: ReactiveTask/Task.swift [hunk 1]
```diff
@@ -252,8 +252,8 @@ private func aggregateDataReadFromPipe(pipe: Pipe) -> SignalProducer<ReadData, R
 				} else {
 					buffer = data
 				}
-			}, error: { error in
-				observer.sendError(error)
+			}, failed: { error in
```
Comment (NachoSoto) [inline ReactiveTask/Task.swift]: You can do: `failed: observer.sendFailed`

**Comment by NachoSoto** [comment]
Nice!

File: ReactiveTask/Task.swift (hunk 1 above)
Comment (ikesyo) [inline ReactiveTask/Task.swift]: Nice catch and done in 4d3fad0! (but I'm not sure about the indentation)

**Comment by ikesyo** [comment]
I'm gonna merge this to cut a new release. :pray:

**Pull request merged**